# request
int8 data
---
# response
bool success
string message